rero/rero-ils
# frontend: cleaning

* NEW: add a npm library for sticky autohide header
* BETTER: add a rule for SASS in editorconfig
* BETTER: extend the sidebar the height
* BETTER: reorder the *my account* items menu
* FIX: fix cover size
* FIX: reimplement navbar inverse colors
* FIX: place correctly main content under the header
* FIX: fix the profile layout
* FIX: fix the frontpage upper section
* FIX: restore the 16px base font-size

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>
Co-Authored-by: Nicolas Labat <nicolas.labat@rero.ch>

---------------------------

## To test

- get the PR branch
- `./scripts/bootstrap` (maybe you'll need setup too, if you still don't have patron type fixtures
- observe the UI
- run the tests

:warning: **There still some issue, obviously, on the general layout. I'm on the process to open an issue to list everything: https://github.com/rero/rero-ils/issues/121**
